services/appflowy-collaborate/src/command.rs

```diff
@@ -50,6 +68,35 @@ pub(crate) fn spawn_collaboration_command(
             },
           }
         },
+        CollaborationCommand::BatchGetEncodeCollab { object_ids, ret } => {
+          if let Some(group_manager) = weak_groups.upgrade() {
+            let tasks = futures::stream::iter(object_ids)
+              .map(|object_id| {
+                let cloned_group_manager = group_manager.clone();
+                tokio::task::spawn(async move {
+                  let group = cloned_group_manager.get_group(&object_id).await;
+                  if let Some(group) = group {
+                    (object_id, group.encode_collab().await.ok())
```

Comment:
it might be an issue when encoding a `big` collab. these futures might run concurrently, so i am thinking about using stream